### DISCUSSION TITLE
How can I use insights app to view the logs of app running on staging mode in vps

### DISCUSSION THREAD

**Discussion by innocent-greats** [serverpod/serverpod#1211] [Q&A] [answered]
<img width="375" alt="serverpod insights error" src="https://github.com/serverpod/serverpod/assets/97280792/3d68b9f5-0ba8-44aa-9a18-8bde2287d700">

How do I use insight up for staging that is running in vps? I am stuck to this error

**Comment by vlidholt** [comment, marked as answer]
Insights connects to a different port than your regular API does. You can configure this in the files in the `config` directory. Insights use the information there to connect to your server.
